integration/airflow/openlineage/airflow/extractors/bigquery_extractor.py

```diff
@@ -60,6 +60,10 @@ def extract_on_complete(self, task_instance) -> Optional[TaskMetadata]:
         stats = BigQueryDatasetsProvider(client=client).get_facets(bigquery_job_id)
         inputs = stats.inputs
         output = stats.output
+
```

Comment:
@mobuchowski currently this method fails when called from a `SqlCheckExtractor`, as the operators do not emit a Job ID. This should be fixed in OSS Airflow.

Reply:
I can make that fix.

Reply:
Actually it seems the operator does do this already on lines 36-49.